h2o-py/h2o/frame.py

```diff
@@ -1093,11 +1093,6 @@ def __getitem__(self, item):
         fr._ex._cache.types = new_types
         return fr
 
-    # def __getattr__(self, key):
```

Comment:
Well, at one point I thought to implement the same functionality they have in pandas: so that for example `frame.C1` can be used to refer to a column "C1" within the frame. I no longer think it's a good idea (and pandas guys probably would agree with me). The problem with this approach is that: (1) this syntax can only be used to access columns that are not any of the existing properties / methods on the H2OFrame (thus the user is expected to know what those 200+ names are), (2) any subsequent change to the frame's API, including adding new methods, will become a breaking change because you are making new names into reserved ones...